packages/wonder-blocks-core/__snapshots__/generated-snapshot.test.js.snap

```diff
@@ -34,6 +34,28 @@ exports[`wonder-blocks-core example 1 1`] = `
   >
     Hello, world!
   </span>
+  <div
+    className=""
+    style={
+      Object {
+        "alignItems": "stretch",
+        "backgroundColor": "goldenrod",
+        "borderStyle": "solid",
+        "borderWidth": 0,
+        "boxSizing": "border-box",
+        "display": "flex",
+        "flexDirection": "column",
+        "margin": 0,
+        "minHeight": 0,
+        "minWidth": 0,
+        "padding": 4,
+        "position": "relative",
+        "zIndex": 0,
+      }
+    }
+  >
+    Even with an array and inline styles!
+  </div>
```

Comment:
Added snapshot because of the docs change. Notice that all other snapshots are unchanged because the added class for inline styles is only done when not snapshotting.